app/regen/cmd/root.go

```diff
@@ -72,7 +72,7 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 				return err
 			}
 
```

Comment:
Do we want to set a minimum-gas-prices on regen? cc @robert-zaremba

Reply:
Let's do "0.025uregen", which is I think what most validators are running on production?